src/screens/CardInfo/CardInfoScreen.tsx

```diff
@@ -38,23 +40,24 @@ export default function CardInfo({ navigation, Card, benefits }: ICardInfoScreen
   proper everything (after api implementation)
   */
 
-  const [screenState, setState] = useState(1);
+  type CardInfoScreenState = 'benefits' | 'business';
+
+  const [screenState, setState] = useState<CardInfoScreenState>('business');
```

Comment:
Nie trzeba robić na to osobnego typu, wystarczy `useState<'benefits' | 'business'>(...)`